packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -33,6 +33,7 @@ import Parser from '../locus-info/parser';
 export default class LocusInfo extends EventsScope {
   constructor(updateMeeting, locus = {}, webex, meetingId) {
     super();
+    this.updateControls(locus.controls);
```

Comment:
Could we move this to ~line 58 with the other `update` functions to keep them grouped together? If not, let's add a comment as to why this is inconsistent.

Reply:
we cannot, so i will add a comment

Reply:
i'll replace this.controls with this.updateControls